### PR TITLE
feat: modal component

### PR DESCRIPTION
**Pull request by alexbarnsley**
## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [ ] Ready to be merged

### CODE REVIEW

File: src/styles/headings.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+h1 {
+	@apply text-2xl font-semibold mb-2;
```
Comment (brenopolanski) [inline src/styles/headings.css:2]: is not `text-4xl` instead of  `text-2xl`?

Comment (alexbarnsley) [reply src/styles/headings.css:2]: There's a media query below this. 2xl is for smaller screens

Comment (alexbarnsley) [reply src/styles/headings.css:2]:
```
@screen md {
	h1 {
		@apply text-4xl;
	}
}
```

Comment (brenopolanski) [reply src/styles/headings.css:2]: oww yeah... I have not seen :P

Comment (alexbarnsley) [reply src/styles/headings.css:2]: 😉

**Pull request merged**